dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/Ds3PanelPresenter.java

```diff
@@ -435,7 +435,7 @@ private void ds3TransferToLocal() {
             }
         }
 
-        final Ds3GetJob getJob = getJobFactory.createDs3GetJob(selectedItemsAtSourceLocationListCustom, localPath);
+        final Ds3GetJob getJob = getJobFactory.createDs3GetJob(selectedItemsAtSourceLocationListCustom, localPath, null);
```

Comment:
Instead of requiring the caller to pass null to this method, overload the method with a new signature that hides the null from the caller.